### PR TITLE
Renamed ParamsScheme to ParamScheme. Added test_init_config_with_non_existing_file to Config tests.

### PR DESCRIPTION
**Pull request by JaviCerveraIngram**
- Fixed a bug in Config where the type of strings was not correctly recognized in Python 2. Fixed using the "six" library, released under MIT license.
- Renamed ParamsScheme to ParamScheme
- Renamed fulfilment_automation.py to fulfillment_automation.py

### CODE REVIEW

**Comment by ht-albert** [comment]
@JaviCerveraIngram  please, actualizing your pull request

File: tests/test_config.py [hunk 1]
```diff
@@ -1,44 +1,60 @@
...
+    _assert_config(config)
+
+
+def _assert_config(config=None):
+    assert Config.api_key == conf_dict.get('apiKey')
+    assert Config.api_url == conf_dict.get('apiEndpoint')
+    assert isinstance(Config.products, list)
+    assert len(Config.products) == 1
+    assert Config.products == [conf_dict.get('products')]
+    if config:
```
Comment (ht-albert) [inline tests/test_config.py]: Config is global
Why send config param in a method `_assert_config`?

Comment (JaviCerveraIngram) [reply tests/test_config.py]: Is it required for any reason that Config has to be global? I'd like to reduce global state as much as possible. In my tests, I am creating a Config instance and working with that.

**Comment by vgrebenschikov** [comment]
Please be a bit more specific in commit message "fixes and small changes" like "something was changed".

Better something like: "Normalize naming, use explicit Config object..."

**Comment by JaviCerveraIngram** [comment]
Sorry about that! Initially this PR contained the fixes and cleanup that I commented on the meeting we had two weeks ago. These changes were made against the code in the old git.int.zone repo. When I applied the changes to the Github version of the code, most of things that I commented on the meeting had already been addressed, so most stuff was removed. I have been on another project for the past two weeks, and after merging upstream changes into my fork, the only things remaning were the renaming of ParamsScheme to ParamScheme and the addition of one test.

I am working on several more additions, so expect another PR probably on Monday.

**Comment by JaviCerveraIngram** [comment]
> xp



> 
> 
> Please be a bit more specific in commit message "fixes and small changes" like "something was changed".
> 
> Better something like: "Normalize naming, use explicit Config object..."

Sorry about that! Initially this PR contained the fixes and cleanup that I commented on the meeting we had two weeks ago. These changes were made against the code in the old git.int.zone repo. When I applied the changes to the Github version of the code, most of things that I commented on the meeting had already been addressed, so most stuff was removed. I have been on another project for the past two weeks, and after merging upstream changes into my fork, the only things remaning were the renaming of ParamsScheme to ParamScheme and the addition of one test. I am working on several more additions, so expect another PR probably on Monday.

**Comment by JaviCerveraIngram** [comment]
> > Please be a bit more specific in commit message "fixes and small changes" like "something was changed".
> > Better something like: "Normalize naming, use explicit Config object..."
> 

Sorry about that! Initially this PR contained the fixes and cleanup that I commented on the meeting we had two weeks ago. These changes were made against the code in the old git.int.zone repo. When I applied the changes to the Github version of the code, most of things that I commented on the meeting had already been addressed, so most stuff was removed. I have been on another project for the past two weeks, and after merging upstream changes into my fork, the only things remaning were the renaming of ParamsScheme to ParamScheme and the addition of one test. I am working on several more additions, so expect another PR probably on Monday.

**Pull request merged**